ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -22,15 +22,11 @@
             rel="stylesheet" />
       <link href="{% static 'css/main.css' %}" rel="stylesheet" />
     {% endblock css %}
-    {% block javascript %}
-      <script defer src="{% static 'js/dist/app.js' %}"></script>
-      <script defer src="{% static 'js/dist/cookie_consent.js' %}"></script>
-    {% endblock javascript %}
     {% include "includes/gtm/gtm_head.html" %}
   </head>
   <body>
     <script>
-      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+      document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
```

Comment:
I don't follow this

Reply:
we now add both the same js-enabled classname, and the govuk-fronted-supported classame, as suggested in [their docs](https://design-system.service.gov.uk/styles/page-template/). The ternary is kinda cryptic, but in general where we have inline scripts i tend to favour brevity so happy with it as is